src/main/java/de/dhbw/karlsruhe/models/User.java

```diff
@@ -1,4 +1,4 @@
-package de.dhbw.karlsruhe.model;
+package de.dhbw.karlsruhe.models;
 
```

Comment:
User Klasse ist in Ordner models aber Difficulty und Sudoku sind noch im Ordner model. Sollte einheitlich sein.

Reply:
Habe alles in das models package gepackt, danke für den Hinweis